Services/DataBaseService/Contollers/DataBaseController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using DTO;
+using DataBaseService.Interfaces;
+
+namespace DataBaseService.Contollers
+{
+    [ApiController]
+    [Route("[controller]")]    
+    public class DataBaseController : ControllerBase
+    {
+        [Route("CreateUser")]
+        [HttpPost]
+        public void CreateUser([FromBody] UserEmailPassword userIn, [FromServices] ICommand<UserEmailPassword> command)
```

Comment:
Why we should name this as userIn and not just user? Do you will have userOut in this method?

Reply:
Ia also think that it is better to send firstly command and then userIn , but it is just a suggestion

Reply:
done